### DISCUSSION TITLE
label-free absolute quantification

### DISCUSSION THREAD

**Discussion by weiclav** [vdemichev/DiaNN#208] [General]
Hi there,

has anybody dealt with label-free absolute quantification using DIA-NN results to get approximate estimates of protein abundances within individual samples? I have found the following publication https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6533198/ and was about to give it a try.

Any plan to implement this feature into DIA-NN in future?

Thanks for any input!

Best wishes,
David

**Comment by vdemichev** [comment]
Hi David,

SILAC is supported. So yes, can do absolute quant with it. As for approximate estimates (with one potential method described in that paper) - these are always fairly imprecise, but can be calculated based on the output of any DIA or DDA tool using R/Python. Built-in support for this in DIA-NN is not planned for now, but obviously can implement it if there's lots of interest.

Best,
Vadim

**Comment by weiclav** [comment]
Hi Vadim,

thanks for a prompt response, really appreciate it!

And would you recommend to use data with the default protein inference algorithm or the alternative one (i.e. --relaxed-prot-inf), I am not decided here yet what might be closer to the reality by design...

Thanks once more!
David

**Comment by vdemichev** [comment]
For this purpose - the default one

**Comment by haianhle** [comment]
@vdemichev Hi Vadim, we are using this version of DIA-NN (v1.8.1 beta 9) and I just want to check with you that SILAC is supported? Thank you so much.

Best wishes,
Anh

**Reply by vdemichev** [reply]
Hi Anh,

Yes, it is, and I would recommend to update to 1.8.1.

Vadim
